vrrpd/vrrp.c

```diff
@@ -651,6 +651,9 @@ void vrrp_vrouter_destroy(struct vrrp_vrouter *vr)
 
 struct vrrp_vrouter *vrrp_lookup(struct interface *ifp, uint8_t vrid)
 {
+	if (!ifp)
+		return NULL;
+
```

Comment:
this sanity check makes me think that vrrp_lookup can be called with null parameter , which is weird.

Reply:
Yeah there were some other code paths in the past that could have called this with a null pointer, I agree this patch looks weird out of context. I can change it to an assert.

Reply:
ok for the assert.
thanks.